ncl/ncl_entries/trigonometric_functions.ipynb

```diff
@@ -0,0 +1,657 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "e630c395-5419-4786-84fe-94bc0f0abbdc",
+   "metadata": {},
+   "source": [
+    "# Trigonometric Functions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cac2fd03-5dcf-478a-8cd5-72207bc2cb4b",
+   "metadata": {},
+   "source": [
+    "# Overview\n",
+    "The trigonometric functions for sine, cosine, and tangent (and associated inverse functions) are built-in functions in Python as well as part of a common Python package `numpy`\n",
+    "\n",
+    "- `math/cmath` vs. `numpy`\n",
+    "- [sine](#sin)\n",
+    "- [cosine](#cos)\n",
+    "- [tangent](#tan)\n",
+    "- [arcsine](#asin)\n",
+    "- [arccosine](#acos)\n",
+    "- [arctangent](#atan)\n",
+    "- [Differences from NCL](#Differences-from-NCL)\n",
+    "- [Python Resources](#Python-Resources)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fd87eed0-c2ea-4b7c-97c4-1f8f0bc22c6a",
+   "metadata": {},
+   "source": [
+    "## Using the `math` and `cmath` module\n",
+    "\n",
+    "The [`math`](https://docs.python.org/3/library/math.html#trigonometric-functions) and [`cmath`](https://docs.python.org/3/library/cmath.html#trigonometric-functions) module are part of the Python Standard Library and require no additional downloading to use. Both math modules can be used to calculate the value of trigonometric functions in Python\n",
+    "\n",
+    "Both math modules expect an input value in radians:\n",
+    "- Input: single input values (float, decimals, integars)\n",
+    "- Returns: single numerical value in radians\n",
+    "\n",
+    "The `math` function only work on real number inputs, while `cmath` contains equivalent functions for for complex and imaginary numbers\n",
+    "\n",
+    "However, if you need to work with lists or arrays neither math module will not be sufficient"
```

Comment:
```suggestion
    "However, if you are working with lists or arrays we suggest using the numpy equivalents"
```

Given that you can work around the list/array limitation this seems like it might skip that rabbit hole and skip to the point.  We can certainly add more detail as well, but that seems like a later problem or if someone asks.